### PR TITLE
Dsl qual branch new

### PR DESCRIPTION
**Pull request by nbhoski**
* Accommodated the MATLAB command as script related changes into the DSL pipeline code.
* Created separate step executor for both command and test steps. 
* Updated the Step definition to  override getDisplayName() for Blue ocean view
* Added help text for both steps

### CODE REVIEW

File: src/main/resources/config.properties [hunk 1]
```diff
@@ -16,4 +16,8 @@ Builder.matlab.modelcoverage.support.warning = To generate a Cobertura model cov
 Builder.matlab.exportstmresults.support.warning = To export Simulink Test Manager results, use MATLAB R2019a or a newer release.
 Builder.matlab.runner.script.target.file.linux.name = run_matlab_command.sh
 Builder.matlab.runner.script.target.file.windows.name = run_matlab_command.bat
-build.workspace.computer.not.found = Unable to access the computer for this build.
\ No newline at end of file
+build.workspace.computer.not.found = Unable to access the computer for this build.
+matlab.command.build.step.name = runMATLABCommand
+matlab.tests.build.step.name = runMATLABTests
+matlab.command.step.display.name = Runs MATLAB command and scripts
+matlab.tests.step.display.name = Runs MATLAB tests and produce test artifacts
```
Comment (nbhoski) [inline src/main/resources/config.properties:23]: @mw-hrastega these are the changes you may review

File: src/main/resources/config.properties [hunk 2]
```diff
@@ -16,4 +16,8 @@ Builder.matlab.modelcoverage.support.warning = To generate a Cobertura model cov
 Builder.matlab.exportstmresults.support.warning = To export Simulink Test Manager results, use MATLAB R2019a or a newer release.
 Builder.matlab.runner.script.target.file.linux.name = run_matlab_command.sh
 Builder.matlab.runner.script.target.file.windows.name = run_matlab_command.bat
-build.workspace.computer.not.found = Unable to access the computer for this build.
\ No newline at end of file
+build.workspace.computer.not.found = Unable to access the computer for this build.
+matlab.command.build.step.name = runMATLABCommand
+matlab.tests.build.step.name = runMATLABTests
+matlab.command.step.display.name = Runs MATLAB command and scripts
```
Comment (mw-hrastega) [inline src/main/resources/config.properties:22]: Run MATLAB commands, scripts, or functions

File: src/main/resources/config.properties (hunk 1 above)
Comment (mw-hrastega) [reply src/main/resources/config.properties:23]: I revised the two statements that you have added. How/where is the text displayed?

File: src/main/resources/config.properties (hunk 2 above)
Comment (nbhoski) [reply src/main/resources/config.properties:22]: Done

File: src/main/resources/config.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (nbhoski) [reply src/main/resources/config.properties:23]: Done

Comment (nbhoski) [reply src/main/resources/config.properties:23]: The text is displayed on Blue ocean pipeline plugin . as shown below 
image

**Pull request merged**